mirrord/kube/src/api.rs

```diff
@@ -81,6 +84,8 @@ pub fn wrap_raw_connection(
                     }
                 }
                 daemon_message = codec.next() => {
+                    timeout_check.reset();
```

Comment:
Shouldn't this be only on DaemonMessage::Pong not on all requests (in case agent got stuck in some loop and is only sending nocensene and doesn't actually respond to Ping with Pong)

Reply:
I don't think we need such validation - the ping/pong is to make sure connection is alive, not for making sure it makes senes